06-Tutorial-CSR.Rmd

```diff
@@ -974,9 +981,96 @@ pool_model_imp <- pool(model_imp)
 summary(pool_model_imp)
 ```
 
-### Tidy {#mistidy}  
+### Tidy
+
+```{r, model1}
+model1 <- cv_total %>%
+  mutate(healthn = as.numeric(health),
+         across(c(educ3, region), as.factor)) %>%
+  lm(formula = healthn ~ men + age + educ3 + region) %>%
+  summary()
+#note: adding lm directly to the pipeline without specifying the parameter's name (formula) will raise an error.
```

Comment:
Explain why. this is because the pipe puts the output (the data) in the first argument of `lm`.